.github/workflows/build.yml

```diff
@@ -0,0 +1,42 @@
+name: CI Build
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    name: Build and Test
+    runs-on: ${{ matrix.os }}
+    # We want to run on external PRs, but not on internal ones as push automatically builds
+    # H/T: https://github.com/Dart-Code/Dart-Code/commit/612732d5879730608baa9622bf7f5e5b7b51ae65
+    if: github.event_name == 'push' || github.event.pull_request.head.repo.full_name != 'amzn/ion-cli'
+    strategy:
+      matrix:
+        # TODO: add windows after fixing build issues
```

Comment:
If the build issue was with `windows-latest`, we can get by using `windows-2019` in the meantime.

Reply:
Tried both `windows-2019` (GH action failure error(s) [here](https://github.com/amzn/ion-cli/runs/5373024441?check_suite_focus=true#step:6:308)) and `windows-latest` (error(s) [here](https://github.com/amzn/ion-cli/runs/5373433476?check_suite_focus=true#step:6:256)) and both seemed to have errors running the CMake step, with some possibly relevant warnings and errors [here](https://github.com/amzn/ion-cli/runs/5373024441?check_suite_focus=true#step:6:642). For what it's worth, also tried blindly updating the `ion-c` submodule to latest commit but failed with similar errors [here](https://github.com/amzn/ion-cli/runs/5372806707?check_suite_focus=true#step:6:308).

This could be issues mentioned in #5 but I do not have enough experience with CMake or Windows builds to be sure.

Reply:
Thanks for giving it a shot. It sounds like that'll take a future deep dive to sort out. We can leave it out for the time being.